File: backend/src/models/Transport.js
const mongoose = require('mongoose');

const transportSchema = new mongoose.Schema({
    name: { type: String, required: true },
    image: { type: String, required: true },
    price: { type: String, required: true },
    location: { type: String, required: true },
});

module.exports = mongoose.model('Transport', transportSchema);

File: java_backend/src/main/java/com/example/demo/model/Transport.java
package com.example.demo.model;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document(collection = "transports")
public class Transport {
    @Id
    private String id;
    private String name;
    private String image;
    private String price;
    private String location;
}
